scripts/utils-openssl.sh

```diff
@@ -40,7 +41,7 @@ install_openssl() {
     if [ ! -d ${OPENSSL_SOURCE_DIR} ]; then
         printf "\tClone OpenSSL ${OPENSSL_TAG} ... "
         git clone --depth=1 -b ${OPENSSL_TAG} ${OPENSSL_GIT} \
-             ${OPENSSL_SOURCE_DIR} &>> $LOG_FILE
+             ${OPENSSL_SOURCE_DIR} 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/utils-openssl.sh line 44:
             ${OPENSSL_SOURCE_DIR} 2>&1 >> $LOG_FILE
             ^-------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                   ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                                           ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```